src/core/ui/components/primitives/avatar.tsx

```diff
@@ -0,0 +1,24 @@
+import { classNames } from '../../../../shared/utils/classnames';
+import { profilePicturePlaceholder } from '../drop-down/icons/profilePicturePlaceholder';
+import styles from './avatar.module.css';
+
+export const Avatar = ({ className, src, onClick }) => {
+  const objectProps: React.ComponentPropsWithoutRef<'object'> = {};
+
+  if (src) {
+    objectProps.data = src;
+    objectProps.type = 'image/jpg';
+  }
+
+  return (
+    <div
+      className={classNames(
+        styles.avatar,
+        onClick && styles.clickableAvatar,
+        className
+      )}
+    >
+      <object {...objectProps}>{profilePicturePlaceholder}</object>
+    </div>
```

Comment:
This will render the profile picture placeholder if no `src` is provided (like having an empty or undefined profile picture) or if the image is not loaded correctly.

Also the Avatar component display a hover picture if `onClick` is provided. This lets us use it both in user mgmt component and in the dropdown...